.github/CONTRIBUTING.md

```
@@ -16,7 +16,7 @@
 
 `data-gtm-click` の一覧に関しては以下を参照。
 
-https://github.com/nekochans/lgtm-cat-ui/blob/main/src/types/gtm.ts
+https://github.com/nekochans/lgtm-cat-frontend/blob/f3212759045512afba9a3ed7130281d4fdf3dc4b/src/utils/gtm.ts#L4
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)